pylint/checkers/variables.py

```diff
@@ -2246,10 +2246,18 @@ def _maybe_used_and_assigned_at_once(defstmt: nodes.Statement) -> bool:
             return True
         if isinstance(value, nodes.Lambda) and isinstance(value.body, nodes.IfExp):
             return True
-        return isinstance(value, nodes.Call) and (
-            any(isinstance(kwarg.value, nodes.IfExp) for kwarg in value.keywords)
-            or any(isinstance(arg, nodes.IfExp) for arg in value.args)
-        )
+        if isinstance(value, nodes.Call):
+            for call in utils.get_all_calls(value):
```

Comment:
We should use ``NodeNG.nodes_of_class(nodes.Call)`` from astroid directly.
```suggestion
            for call in value.nodes_of_class(klass=nodes.Call):
```

Reply:
Thanks for pointing this out!